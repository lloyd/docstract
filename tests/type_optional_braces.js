/**
 * when defining the @type of a property, you chose to omit braces
 */

/**
 * @property foo
 * @type {string}
 */

/**
 * @property bar
 * @type string
 */
