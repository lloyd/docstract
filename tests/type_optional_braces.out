{
  "desc": "when defining the @type of a property, you chose to omit braces", 
  "filename": "tests/type_optional_braces.js", 
  "module": "type_optional_braces", 
  "properties": [
    {
      "name": "foo", 
      "source_lines": [
        5, 
        9
      ], 
      "type": "string"
    }, 
    {
      "name": "bar", 
      "source_lines": [
        10, 
        13
      ], 
      "type": "string"
    }
  ], 
  "source_lines": [
    1, 
    4
  ]
}
